lib/spoom/cli/commands/metrics.rb

```diff
@@ -0,0 +1,26 @@
+# typed: true
+# frozen_string_literal: true
+
+require_relative '../../snapshot'
+# require_relative '../../timeline'
+require_relative '../command_helper'
+
+module Spoom
+  module Cli
+    module Commands
+      class Metrics < Thor
+        include Spoom::Cli::CommandHelper
+
+        default_task :snapshot
+
+        desc "snapshot", "run srb tc and display metrics"
+        def snapshot
+          in_sorbet_project!
+
+          snapshot = Spoom::Snapshot.snapshot
```

Comment:
Why is this class called `Snapshot`? It seems to be representing `Metrics` to me.

Reply:
Metrics generally means something you can count but I also want to store things like version numbers for example.

The next pull-request introduce the concept of `Timeline`, each set of data collected of a commit is a `Snapshot`.

Also it was confusing between `Spoom::Metrics` (the snapshot) and `Spoom::Sorbet::Metrics` (the metrics produced by Sorbet).

Reply:
WDYT about `MetricSnapshot` or `MetricsSnapshot`. It more clearly defines what this is a snapshot of.

Reply:
This will become clearer as I add data to this thing in the following pull-requests.